contact.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/dist/output.css" />
+    <title>Baumeister</title>
+  </head>
+  <script>
```

Comment:
it's generally a good practice to place your scripts at the end of the <body> tag to ensure that the HTML elements are loaded before the script is executed.